templates/repo/issue/labels/label_list.tmpl

```diff
@@ -33,6 +33,11 @@
 		<li class="item">
 			<div class="label-title">
 				{{RenderLabel $.Context .}}
+				{{if not .ArchivedUnix.IsZero}}
+					<span class="gt-ml-3 gt-cursor-default gt-italic" data-tooltip-content="{{$.locale.Tr "repo.issues.label_archive_tooltip"}}">
+						{{$.locale.Tr "home.show_archived"}}
```

Comment:
No idea why it's named "show_archived", but it might be fine.

Reply:
I just want a `Archived` translation in US_EN. So I reused it. Rather than creating an alias in locals.

Reply:
But it might be wrong in the future.

`show_archived` might be changed to "Show Archived" ?

Reply:
No worries, happy to add new `archived` in locals.

Reply:
ps: it's not a blocker, just a question , in case some people really like to reuse the exiting one.

(I just recalled that such thing happened in history, like "color" or something, the shared key was used for different meanings, then it broke one day ....)

Reply:
I agree with your point.